karapace/karapace.py

```diff
@@ -22,20 +22,21 @@
 
 
 class KarapaceBase(RestApp):
-    def __init__(self, config_file_path: str, config: dict) -> None:
+    def __init__(self, config: dict) -> None:
+        sentry_config = config.get("sentry", {"dsn": None}).copy()
+        super().__init__(app_name="karapace", sentry_config=sentry_config)
```

Comment:
Not strictly necessary, but it is always easier to reason about these things if `super().__init__` is always done at start or end.